extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RuleAuthorization.scala

```diff
@@ -42,6 +42,7 @@ class RuleAuthorization(spark: SparkSession) extends Rule[LogicalPlan] {
 object RuleAuthorization {
 
   val KYUUBI_AUTHZ_TAG = TreeNodeTag[Boolean]("__KYUUBI_AUTHZ_TAG")
+  val PERMANNENT_VIEW_SUBQUERY_TAG = TreeNodeTag[Boolean]("__PERMANNENT_VIEW_SUBQUERY_TAG")
```

Comment:
TreeNodeTag[Unit] might be better, using `__KYUUBI` prefix to avoid potential naming contention with spark

Reply:
Done